src/tmpfiles/tmpfiles.c

```diff
@@ -3678,6 +3694,7 @@ static int parse_line(
         i.allow_failure = allow_failure;
         i.try_replace = try_replace;
         i.purge = purge;
+        i.ignore_if_target_missing = ignore_if_target_missing;
```

Comment:
This should be refused for commands other than L. Please introduce a similar check as needs_purge() below